### PR TITLE
Fixes for uottawa.brightspace.com

### PR DESCRIPTION
**Pull request by GentlyTech**
- Fixed uOttawa logo on homepage
    - (amended) ~~Used filter in CSS instead of INVERT because the changes should only apply when dark~~ Just used normal INVERT as requested
- Fixed dropdown backgrounds
- Fixed course section cards on homepage
- Fixed borders on some buttons and headers
- Fixed certain button backgrounds on hover/focus

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -21293,6 +21293,45 @@ CSS
 
 ================================
 
+uottawa.brightspace.com
+
+CSS
+button:hover, button:focus, :host([active]) button {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:21299]:
```suggestion
button
:host([active]) button {
```

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -21293,6 +21293,45 @@ CSS
...
+.d2l-label-text:has(.d2l-button-subtle-content):active {
+    background-color: var(--darkreader-bg-d2l-color-gypsum) !important;
+}
+.d2l-navigation-centerer {
+    color: inherit;
+}
+.d2l-navigation-link-image-container {
+    filter: invert(1);
+    mix-blend-mode: lighten;
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:21316]: Hmm this doesn't look sound, what does this specifically fix?

**Review by Gusted — CHANGES_REQUESTED** [review]
Please add `!important;` after the CSS declarations.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (GentlyTech) [reply src/config/dynamic-theme-fixes.config]: Hmm, removing the hover/focus selectors makes the changes not apply to the button when you hover over it.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config]: The `!important` on the CSS declaration should take care of that..

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (GentlyTech) [reply src/config/dynamic-theme-fixes.config:21317]: The filter and mix-blend-mode is for the university logo in the nav bar. I tried using invert but it wouldn't look quite right.

However I did realize a problem with this (apart from the slight "jankiness" of it), which is that you can't see the logo if you set the scheme to light.

So I guess just remove it for now?

Fix with INVERT:

WithINVERT

Fix with CSS:

WithCSS

When using light scheme (when applying the CSS):

WithCSS_Light

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (GentlyTech) [reply src/config/dynamic-theme-fixes.config]: Oh, changing the selector to :host works (the site has an obsession with shadow dom).

Comment (GentlyTech) [reply src/config/dynamic-theme-fixes.config]: Ok, nonetheless I have added the `!imporatant` statements. Sorry about that.

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config]: Yeah normally we just do `INVERT` and _accept_ the jankiness.

Comment (GentlyTech) [reply src/config/dynamic-theme-fixes.config]: Sorry by jankiness I was referring to my CSS patch. But ok, will just do invert.

**Comment by GentlyTech** [comment]
Ok I have updated the commit with your requested changes.

**Review by Gusted — APPROVED** [review]
Thanks.

**Pull request merged**